segment-anything/checkpoints/README.md

```diff
@@ -0,0 +1,3 @@
+# SAM Checkpoints
+
+Download the checkpoints from [here](https://github.com/facebookresearch/segment-anything#model-checkpoints) and place them in this directory.
```

Comment:
Ok, but why is there no matching .gitignore so that the checkpoints do not get commited by accident?